### PR TITLE
os/sdk: document using cork as an option

### PR DESCRIPTION
**Pull request by euank**
cc @crawford

### CODE REVIEW

**Comment by bgilbert** [comment]
Might be good to document that `cork create` and `cork enter` want to sudo.

File: os/sdk-modifying-coreos.md [hunk 1]
```diff
@@ -28,7 +27,40 @@ git config --global user.name "Your Name"
...
+
+You may want to add the `PATH` export to your shell profile (e.g. `.bashrc`).
+
+
+Next, the cork utility can be used to create an SDK chroot:
+
+```sh
+mkdir coreos-sdk
+cd coreos-sdk
+cork create --sdk-version=master
```
Comment (marineam) [inline os/sdk-modifying-coreos.md]: Don't pass `--sdk-version=master` the, the latest SDK release is detected from the manifest git repository automatically. I'm not sure what "master" is pointing at but in general only official release builds should be used.

File: os/sdk-modifying-coreos.md [hunk 2]
```diff
@@ -28,7 +27,40 @@ git config --global user.name "Your Name"
...
+cork enter
+```
+
+To use the SDK chroot in the future, run `cork enter` from the above directory.
+
+### Option 2: Manually
```
Comment (marineam) [inline os/sdk-modifying-coreos.md]: I wouldn't exactly call this manual, it is more of just changing the order in which the SDK is set up. Maybe call it "Using repo and cros_sdk"

Comment (marineam) [inline os/sdk-modifying-coreos.md]: Similarly, references to `cros_sdk` below should be moved under this header.

**Comment by euank** [comment]
Addressed comments.

File: os/sdk-modifying-coreos.md [hunk 3]
```diff
@@ -67,6 +102,23 @@ Synchronize all of the required git repos from the manifest.
...
+```sh
+./chromite/bin/cros_sdk
+```
+
+**WARNING:** To delete the SDK chroot, use `./chromite/bin/cros_sdk --delete`. Otherwise, you will delete `/dev` entries that are `bind`-mounted into the chroot.
+
+Set up user `core`'s password:
+
+```sh
+./set_shared_user_password.sh
```
Comment (marineam) [inline os/sdk-modifying-coreos.md]: This is applicable regardless of how you enter the chroot.

**Pull request merged**